src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -351,3 +355,36 @@ def body(self):
             JMESPathCheck('[0].name', self.webapp + '/' + slot),
             ])
         self.cmd('appservice web deployment slot delete -g {} -n {} --slot {}'.format(self.resource_group, self.webapp, slot), checks=NoneCheck())
+
+class WebappSSLCertTest(ResourceGroupVCRTestBase):
+
+    def __init__(self, test_method):
+        super(WebappSSLCertTest, self).__init__(__file__, test_method, resource_group='azurecli-webapp-ssl')
+        self.webapp_name = 'webapp-ssl-test123'
+
+    def test_webapp_ssl(self):
+        self.execute()
+
+    def body(self):
+        plan = 'webapp-ssl-test'
+
+        #Cert Generated using
+        #https://docs.microsoft.com/en-us/azure/app-service-web/web-sites-configure-ssl-certificate#bkmk_ssopenssl
+
+        pfx_file = os.path.join(TEST_DIR, 'server.pfx')
+        cert_password = 'test'
+        cert_thumbprint = 'DB2BA6898D0B330A93E7F69FF505C61EF39921B6'
+        self.cmd('appservice plan create -g {} -n {} --sku B1'.format(self.resource_group, plan))
+        self.cmd('appservice web create -g {} -n {} --plan {}'.format(self.resource_group, self.webapp_name, plan))
```

Comment:
Do these calls take a long time?  If so, consider moving them into the set_up method. That what they are skipped when the recording is played back. If they don't take a long time (<1 minute) then it's probably fine to leave them in.